tailwind.config.js

```diff
@@ -1,10 +1,18 @@
 module.exports = {
-  content: [
-    "./pages/**/*.{js,ts,jsx,tsx}",
-    "./components/**/*.{js,ts,jsx,tsx}",
-  ],
+  content: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
   theme: {
-    extend: {},
+    extend: {
+      backgroundImage: {
+        shadow: 'linear-gradient(to bottom, rgba(0, 0, 0, 0.2) 0%, rgba(0, 0, 0, 1) 100%)',
+      },
+      fontFamily: {
+        neuzeit: '"Neuzeit S LT Std"',
+      },
+      fontSize: {
+        baseLarge: ['17px', '22px'],
+        baseSmall: ['15px', '20px'],
```

Comment:
```suggestion
        'base-large': ['17px', '22px'],
        'base-small': ['15px', '20px'],
```

This is a little bit tricky because these javascript variables are also the name for CSS classes :D So should we use the JavaScript naming conventions or the CSS ones? 
I would recommend the CSS one in order to keep components nice and clean 😇

Reply:
sure, renamed in https://github.com/tyrro/react-survey/pull/50/commits/0534d29c922369266f9200ad61515222e6e7a9f8